osm_cli/converters/pdf_converter.py

```diff
@@ -0,0 +1,62 @@
+import socket
+
+import docker
+from docker.errors import DockerException
+from pydantic import FilePath
+import requests
+
+from osm_cli.converters.converter import Converter
+from osm_cli.utils.config import config
+
+
+class PDFConverter(Converter):
+    protocol: str = config.PROTOCAL
```

Comment:
Is this intentionally PROTOCAL? Change if not.

Reply:
We have now updated it accordingly.